### DISCUSSION TITLE
Global string setting to replace "Unknown" or "Unavailable" values

### DISCUSSION THREAD

**Discussion by GizmoGoody** [home-assistant/frontend#12757] [Configuration]
It's unavoidable that some sensors become unknown or unavailable. When that happens, one of those 7 to 11 character strings takes the place of a 1-3 character string (e.g. a battery level %) usually disrupting the visual depiction of that card/element. I would like an ability to set a global string that would take the place of those default strings in situations like these.

For example, instead of Unknown, I might set "?" and instead of Unavailable, a "!"

Of course, people could leave the default values if that's what they prefer. I know I could put these as additional conditional statements in my templates, but something like what I describe above seems like a better solution.

**Comment by BoneheadFraggle** [comment]
I like that, and would like to add another option, to make it just "blank".

I would like the option to have my dashboards show 
"Humidity Living room          %" 
rather than today's 
"Humidity Livingroom   NaN %"

I think it is visually nicer, and you still get the possibility to fetch unavailable sensors by triggering on "not greater than -100"

**Reply by GizmoGoody** [reply]
<kind voice>If you like this idea, I think you're also supposed to Upvote it.</kind voice> ;-)
